backup/v1.2.1/ui/main_window.py

```
@@ -0,0 +1,1100 @@
+                self.save_config('last_output_folder', folder)
+                # Log to debug file
+                log_file = get_resource_path('log.txt')
+                with open(log_file, 'a', encoding='utf-8') as log:
+                    log.write(f"[{datetime.now()}] MainWindow set_output_folder called: Saving '{folder}' to config\n")
+            except Exception as e:
+                # Log error
+                try:
+                    log_file = get_resource_path('log.txt')
+                    with open(log_file, 'a', encoding='utf-8') as log:
+                        log.write(f"[{datetime.now()}] Error saving folder from MainWindow: {str(e)}\n")
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)